index.html

```diff
@@ -0,0 +1,130 @@
+</head>
+<body>
+  <div class="background-container">
+  <header id="header">
+    <button type="button" id="open" class="btn-open" onclick="openMenu()"><i class="uil uil-bars"></i></button>
+    <nav class="top-header">
+      <ul class="list-container-one">
+        <li><a href="#"><i class="uil uil-facebook-f"></i></a></li>
+        <li><a href="#"><i class="uil uil-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </nav>
+    <div class="bottom-header">
+      <nav class="nav-container" id="nav">
+        <button type="button" class="btn-close btn" onclick="closeMenu()">X</button>
+        <ul class="list-container-two">
+          <li><a class="close btn-close" href="about.html">About</a></li>
+          <li><a class="close btn-close" href="#">Program</a></li>
+          <li><a class="close btn-close" href="#">Join</a></li>
+          <li><a class="close btn-close" href="#">Sponsor</a></li>
+          <li><a class="close btn-close" href="#">News</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="bottom-header-desktop">
+      <a class="logo-link" href="index.html"><img class="logo-img" src="img/logo.png"><a/>
+      <nav class="nav">
+        <ul class="list-container-desktop">
+          <li><a class="close btn-close" href="about.html">About</a></li>
+          <li><a class="close btn-close" href="#">Program</a></li>
+          <li><a class="close btn-close" href="#">Join</a></li>
+          <li><a class="close btn-close" href="#">Sponsor</a></li>
+          <li><a class="close btn-close" href="#">News</a></li>
+        </ul>
+      </nav>
+      <button type="button" class="btn-CC">CC Campaign</button>
+    </div>
+  </header>
+  <section id="home">
+    <div class="first-description">
+      <p class="hello">"Hello! Sharing world"</p>
+      <h2 class="creative">KAL TECH THINK UNLIMITED 2023</h2>
+    </div>
+    <div class="second-description">
+      <p class="paragraf">Learn the fundamentals of web development and build your own web pages using HTML, CSS, and JavaScript. This course will provide you with the necessary foundations to start your career in the world of web development.</p>
+    </div>
+    <div class="three-description">
+      <h3 class="date">2023.05.24(THU) ~ 26(FRI)</h3>
+      <p class="adrees">@ "Learn Web Development: Practical and Up-to-Date Courses"</p>
+    </div>
+  </section>
+  </div>
+  <section id="main-program">
+    <h2>Main Program</h2>
+    <h2 class="line">______</h2>
+    <div class="cards">
+      <div class="card box1">
+        <i class="fa-brands fa-html5"></i>
+        <h4>HTML</h4>
+        <p>HTML (HyperText Markup Language) is a markup language used to create the structure and content of web pages.</p>
+      </div>
+      <div class="card box2">
+        <i class="fa-brands fa-css3-alt"></i>
+        <h4>CSS</h4>
```

Comment:
- [ ] For this ssection, kindly make adjustment to the spacing as follows:
- The first header section and the second black section
- IInbetween the `main` heeader eriting and the boxes
- Kindly change the color of the boxes to grey just as in the original design. Same thing should be done to mobile 👍 

## Original design:
![image](https://github.com/Kal2403/capstone-project/assets/50990925/d763219b-a8f1-4a9a-b527-2da1b559f8bf)

## Your work:
![image](https://github.com/Kal2403/capstone-project/assets/50990925/fcf4e9e0-1cdd-47ec-887e-fd3a0e05420c)